### PR TITLE
fix: trigger export when saving destination data

### PR DESCRIPTION
**Pull request by pedroslopez**

### CODE REVIEW

File: core/src/actions/destinations.ts [hunk 1]
```diff
@@ -189,6 +190,8 @@ export class DestinationEdit extends AuthenticatedAction {
       destination.groupId
     ) {
       await destination.unTrackGroup();
+    } else if (params.triggerExport) {
+      await destination.exportGroupMembers();
```
Comment (pedroslopez) [inline core/src/actions/destinations.ts]: Should this be forced?

Comment (pedroslopez) [reply core/src/actions/destinations.ts]: (yes, nothing is actually exported if it's not forced)

Comment (evantahler) [reply core/src/actions/destinations.ts]: Yep - because it will look like all the exports have no changes otherwise.

**Review by evantahler — APPROVED** [review]
Thanks for changing this @pedroslopez, and I'm sorry I didn't catch this yesterday.

Confirming the behavior here:
* Back to one button on the page
* That button will always create a run, regardless of if the group being tracked has changed or not.

In another story (https://www.pivotaltracker.com/story/show/178055151) we can decide if we want to not enqueue the run if there are no changes to the destination

**Pull request merged**